frontend/src/scenes/billing/CreditCTAHero.tsx

```diff
@@ -15,12 +15,12 @@ export const CreditCTAHero = (): JSX.Element | null => {
     const { width, ref: heroRef } = useResizeObserver()
     const { featureFlags } = useValues(featureFlagLogic)
 
-    const { creditOverview, isPurchaseCreditsModalOpen, isCreditCTAHeroDismissed, computedDiscount } =
+    const { creditOverview, isPurchaseCreditsModalOpen, isCreditCTAHeroDismissed, computedDiscount, isManagedAccount } =
         useValues(billingLogic)
     const { showPurchaseCreditsModal, toggleCreditCTAHeroDismissed } = useActions(billingLogic)
 
     const isEligible = creditOverview.eligible || featureFlags[FEATURE_FLAGS.SELF_SERVE_CREDIT_OVERRIDE]
-    if (creditOverview.status === 'paid' || !isEligible) {
+    if (isManagedAccount || creditOverview.status === 'paid' || !isEligible) {
```

Comment:
I'd rather this check go in billing for credit overview logic so it stays all together.

Reply:
Do you mean to include `is_managed_account` in the `/billing` API response (derived from `account_owner`)? Or to incorporate it into `credits_overview` returned from `/billing (either by influencing `eligible` or as a separate property on `credit_overview` ?

Reply:
Add the check for if it's a managed account into `credits_overview` (https://github.com/PostHog/billing/blob/main/billing/models/customer.py#L2565). 

So it they are a managed account return eligible as false.